diff --git a/node_modules/react-native-app-settings/android/build.gradle b/node_modules/react-native-app-settings/android/build.gradle
index 849e79a..f87bfa7 100644
--- a/node_modules/react-native-app-settings/android/build.gradle
+++ b/node_modules/react-native-app-settings/android/build.gradle
@@ -31,6 +31,6 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
   
\ No newline at end of file
